crates/foundry-ui/src/components/styled.rs

```diff
@@ -0,0 +1,26 @@
+use console::Style;
+use serde::Serialize;
+
+use crate::Message;
+
+/// Generic textual message with applied style.
+#[derive(Serialize)]
+pub struct StyledMessage<'a> {
+    text: &'a str,
+    #[serde(skip)]
+    text_style: &'a str,
+}
+
+impl<'a> StyledMessage<'a> {
+    #[must_use]
+    pub fn new(text: &'a str, text_style: &'a str) -> Self {
+        Self { text, text_style }
+    }
+}
+
```

Comment:
Not sure about this struct. In what cases do we want to style the whole message?

Reply:
I will remove it completely, was meant to be used somewhere but finally wasn't.